ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/Conversions.scala

```diff
@@ -268,6 +294,159 @@ private[state] object Conversions {
       }.toMap,
     )
 
+  def decodeTransactionRejectionEntry(
+      entry: DamlTransactionRejectionEntry
+  ): Option[FinalReason] = {
+    def buildStatus(code: Code, message: String) = {
+      Status.of(
+        code.value,
+        message,
+        Seq.empty,
+      )
+    }
+
+    val status = entry.getReasonCase match {
+      case DamlTransactionRejectionEntry.ReasonCase.INVALID_LEDGER_TIME =>
+        val rejection = entry.getInvalidLedgerTime
+        Some(
```

Comment:
Maybe pull out `Some(…)` here.

Reply:
There's one None so not sure how to make the compiler happy without this.

Reply:
Sorry, must have missed that.